### PR TITLE
[TAO-7780] Tutor - allow for additional candidates on proctor screen

### PR DESCRIPTION
**Pull request by AntonTsymuk**
Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/TAO-7780
 
Add a control to the datatable component to allow users to select amount of items per page

  
#### How to test

- login as a Proctor user
- navigate to Monitor page
- try to change amount of items per page
  
#### Dependencies
 
Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-proctoring/pull/675

### CODE REVIEW

**Comment by zagovorichev** [comment]
@AntonTsymuk Correct me, pls, if I'm wrong, but I think:
1. we do need qunit tests for the new feature (I guess here https://github.com/oat-sa/tao-core/tree/6a9a9eff588ab9802354a47ef288da3efb645d13/views/js/test/ui/datatable)
2. I think that this code could be moved to the new component, just for selecting some data. What I mean: we can add a container on the top right/bottom of the table and use as many components as we wish for the same page.

File: views/js/ui/datatable.js [hunk 1]
```diff
@@ -53,7 +54,15 @@ define([
...
-        }
+        },
+        pageSizes: [
+            { label: '25 ' + __('items per page'), selected: true, value: 25 },
+            { label: '50 ' + __('items per page'), value: 50 },
+            { label: '75 ' + __('items per page'), value: 75 },
+            { label: '100 ' + __('items per page'), value: 100 },
+            { label: '200 ' + __('items per page'), value: 200 },
+        ],
+        pageSizeSelector: false,
```
Comment (zagovorichev) [inline views/js/ui/datatable.js]: for the 200 rows case, it could be helpful to have the same pageSizer on the bottom of the table

Comment (AntonTsymuk) [reply views/js/ui/datatable.js]: the selector has been extracted into separated component

Comment (AntonTsymuk) [reply views/js/ui/datatable.js]: the component has been extracted to separated component

File: views/js/ui/pageSizeSelector.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    /**
+     * Builds a select component with page size options
+     *
+     * @param {Object} config
+     * @param {Number} [config.selectedSize] - selected page size
+     * @param {Object} [config.items] - available options
+     * @returns {pageSizeSelector}
+     */
+    function pageSizeSelectorFactory(config) {
```
[the remark this answers is not in the document]
Comment (zagovorichev) [reply views/js/ui/pageSizeSelector.js]: I guess it should be changed to:
`var pageSizeSelectorFactory = function pageSizeSelectorFactory `

**Pull request merged**